internal/storage/ls/ls.go

```diff
@@ -0,0 +1,167 @@
+package ls
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/storage/client"
+	"github.com/supabase/cli/internal/utils"
+)
+
+const STORAGE_SCHEME = "ss"
```

Comment:
This should live in global scope of the package, as it's tied to the whole `storage` concept, not just `ls`.